app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -1,10 +1,48 @@
 import React from "react"
-
-const ApartmentIndex = () => {
+import {Card, CardBody,CardText, CardTitle, ListGroup, ListGroupItem, CardLink} from "reactstrap"
+const ApartmentIndex = ({mockApartments}) => {
+  
   return (
-    <>
-      <h3>ApartmentIndex</h3>
-    </>
+    <div>
+    {mockApartments.map((mockApartment)=> {
+      return ( 
+        <>
+        <Card style={{width: '18rem'}}>
```

Comment:
Card and the react fragment are doing the same job of returning one item.

Reply:
Removed!